drivers/backup/portworx/portworx.go

```diff
@@ -1113,6 +1113,33 @@ func (p *portworx) GetBackupUID(ctx context.Context, backupName string, orgID st
 	return "", fmt.Errorf("backup with name '%s' not found for org '%s'", backupName, orgID)
 }
 
+func (p *portworx) GetRecentScheduleBackupName(ctx context.Context, backupName string, orgID string) (string, error) {
```

Comment:
Make it `backupScheduleName`

Reply:
I am considering renaming backupName to simply scheduleName, as the latter is more consistent with the UI.